packages/contracts/contracts/optimistic-ethereum/libraries/constants/Lib_PredeployAddresses.sol

```diff
@@ -5,14 +5,14 @@ pragma solidity >0.5.0 <0.8.0;
  * @title Lib_PredeployAddresses
  */
 library Lib_PredeployAddresses {
-    address public constant ovmL2ToL1MessagePasser = 0x4200000000000000000000000000000000000000;
-    address public constant ovmL1MessageSender = 0x4200000000000000000000000000000000000001;
-    address public constant ovmDeployerWhitelist = 0x4200000000000000000000000000000000000002;
-    address public constant ovmECDSAContractAccount = 0x4200000000000000000000000000000000000003;
-    address public constant ovmSequencerEntrypoint = 0x4200000000000000000000000000000000000005;
-    address public constant ovmETH = 0x4200000000000000000000000000000000000006;
-    address public constant ovmL2CrossDomainMessenger = 0x4200000000000000000000000000000000000007;
-    address public constant libAddressManager = 0x4200000000000000000000000000000000000008;
-    address public constant ovmProxyEOA = 0x4200000000000000000000000000000000000009;
-    address public constant ERC1820Registry = 0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24;
+    address constant L2_TO_L1_MESSAGE_PASSER = 0x4200000000000000000000000000000000000000;
+    address constant L1_MESSAGE_SENDER = 0x4200000000000000000000000000000000000001;
+    address constant DEPLOYER_WHITELIST = 0x4200000000000000000000000000000000000002;
+    address constant ECDSA_CONTRACT_ACCOUNT = 0x4200000000000000000000000000000000000003;
+    address constant SEQUENCER_ENTRYPOINT = 0x4200000000000000000000000000000000000005;
+    address constant OVM_ETH = 0x4200000000000000000000000000000000000006;
+    address constant L2_CROSS_DOMAIN_MESSENGER = 0x4200000000000000000000000000000000000007;
+    address constant LIB_ADDRESS_MANAGER = 0x4200000000000000000000000000000000000008;
+    address constant PROXY_EOA = 0x4200000000000000000000000000000000000009;
+    address constant ERC1820_REGISTRY = 0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24;
```

Comment:
I changed the case to this, which I think is more in line with 'standard' solidity.